### DISCUSSION TITLE
v4.0.3

### DISCUSSION THREAD

**Discussion by Moonbase59** [Moonbase59/autocue#24] [Announcements]
### 2024-06-18 – v4.0.3

- Changed default of `-x`/`--extra` and `settings.autocue.cue_file.overlay_longtail` from `-15.0` LU to `-12.0` LU, requested by @RM-FM and the community. Together with the `-d`/`--drop` default change from `60.0` to `40.0`, this makes for a "tighter" playout and doesn’t lose too much of long or sustained endings.
- Updated README and Presentation as well, to reflect the new default.

<hr /><em>This discussion was created from the release <a href='https://github.com/Moonbase59/autocue/releases/tag/v4.0.3'>v4.0.3</a>.</em>

**Comment by chourmovs** [comment]
This one work pretty well on my nested alpine virtualbox azuracast instance
Keep up the good work

**Reply by Moonbase59** [reply]
Good to know, thanks!
